### ISSUE TITLE
remap_stats: wrong stat id's getting assigned after hard traffic_server crash

### ISSUE THREAD

**Issue by traeak** [apache/trafficserver#6237]
Recreated with 0cf7e5953aec06d4dafb183861afe918763cbf33

Using the remap_stats plugin in production we have observed the remap_stats::stats_add() TSStatsCreate() getting the id from another stats.  This happens only after a hard ATS crash and restart by traffic_manager.  We've never seen an invalid id assigned, just an id from another stat.  This may result in a bytes transferred counter incrementing a status_5xx like so (from *traffic_ctl metric match remap_stats*)

```
...
82704 plugin.remap_stats.foo1.status_2xx
727229625 plugin.remap_stats.loopback.status_5xx
```

We've found that making the remap_stats persistent (param '-p') seems to mitigate whatever race condition there is.

**Comment by bryancall** [comment]
This is likely resolved with the removal of traffic_manager in ATS 10. Closing.

If this is still an issue, please reopen or file a new issue with reproduction steps.
